### ISSUE TITLE
Text version of e-mail has weird encoding

### ISSUE THREAD

**Issue by mattiasgeniar** [spatie/mailcoach-support#62]
I know, you're probably already sighing with frustration and anger just by that title alone. Text & encoding issues? This is going to be a fun one! 🙈

When supplying a text/plain version to the `SendMailAction`, it seems to do a 7-bit (`Content-Transfer-Encoding: quoted-printable`) encoding of the content. It makes it look like this.

```
--_=_swift_1581452175_cc5ec0e567d5069ddcac9828c0c6ac9f_=_
Content-Type: text/plain; charset=utf-8
Content-Transfer-Encoding: quoted-printable

Hi everyone! =F0=9F=91=8B

[...]

Enjoy this issue! =E2=98=95=EF=B8=8F
```

When using the same input, Sendy sends the mail like this.

```
--b1_b43aeea7fa1bfffc7ae5251fb52c76cf
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Hi everyone! 👋

[...]

Enjoy this issue! ☕️
```

From what I can tell, default Laravel mailables are being used in Mailcoach. There seems to be an option to overwrite the transport layer in `SwiftMailer` and replace it with 8bit instead of the default 7bit (=`quoted-printable`). It's poorly documented though: https://swiftmailer.symfony.com/docs/messages.html#setting-the-encoding

For the life of me, I can't get this going. `SwiftMailer` has no namespacing (wth?) and I can't get those functions to work.

Since you're Laravel experts: what is the proper way to get the text/plain version in the mail to take my text input _as-is_ and not re-encode it?

And should you consider 8bit the default for sending text over 7bit?

**Comment by freekmurze** [comment]
We have decided to revoke your license.

**Comment by freekmurze** [comment]
Just kidding, we'll look into this tomorrow. 👍

**Comment by freekmurze** [comment]
Working on a fix right now...

**Comment by freekmurze** [comment]
This has been fixed in the newly released `v1.6.11`.

**Comment by mattiasgeniar** [comment]
Confirmed, it works!
